docs/src/examples/jump_umap_analysis_with_cosmicqc.py

```diff
@@ -0,0 +1,421 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.16.2
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# # JUMP UMAP analysis with coSMicQC
+#
+# This notebook analyzes [JUMP](https://jump-cellpainting.broadinstitute.org/) data by leveraging [UMAP](https://arxiv.org/abs/1802.03426) and [coSMicQC](https://github.com/WayScience/coSMicQC).
+#
+# ## Outline
+#
+# We focus on a single file from the JUMP dataset: [`BR00117012.sqlite`](https://open.quiltdata.com/b/cellpainting-gallery/tree/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00117012/BR00117012.sqlite).
+# This file is downloaded and prepared by [CytoTable](https://github.com/cytomining/CytoTable) to form a single-cell [Parquet](https://parquet.apache.org/) file which includes all compartment feature data.
+# We use coSMicQC to find and remove erroneous outlier data in order to prepare for UMAP analysis.
+# Afterwards, we use UMAP to demonstrate patterns within the data.
+#
+
+# + editable=true slideshow={"slide_type": ""}
```

Comment:
Is this an artifact of something or is this needed?

Reply:
Thanks, this appears due to how the notebook is automatically exported using [jupytext](https://jupytext.readthedocs.io/en/latest/) (which saves time and reduces human-error from manually exporting the files separately). I'll see if there's a way to address this, but generally what's there is due to how the data are exported from JSON form.